spec/Section 6 -- Execution.md

```diff
@@ -310,12 +310,12 @@ MapSourceToResponseEvent(sourceStream, subscription, schema, variableValues):
 
 - Let {responseStream} be a new _event stream_.
 - When {sourceStream} emits {sourceValue}:
-  - Let {response} be the result of running
+  - Let {executionResult} be the result of running
     {ExecuteSubscriptionEvent(subscription, schema, variableValues,
     sourceValue)}.
   - If internal {error} was raised:
     - Cancel {sourceStream}.
-    - Complete {responseStream} with {error}.
+    - Complete {executionResult} with {error}.
```

Comment:
This should be unchanged - on an internal error we terminate `responseStream` with the error:

```suggestion
    - Complete {responseStream} with {error}.
```